contents/java/JUnit5.md

```diff
@@ -0,0 +1,243 @@
+<frontmatter>
+  title: Introduction to JUnit 5
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Testing in Java with JUnit 5
```

Comment:
Are the benefits only specific to JUnit *5*? Could this article be called **Introduction to Unit Testing in Java with JUnit**?

Reply:
Yes. This guide actually uses things specific to JUnit 5, such as the Extensions model.